packages/manager/apps/web-hosting/src/pages/dashboard/ssl/Ssl.page.tsx

```diff
@@ -1,30 +1,13 @@
-import React, { useState } from 'react';
+import React from 'react';
 
-import { OdsMessage } from '@ovhcloud/ods-components/react';
-import { ODS_MESSAGE_COLOR } from '@ovhcloud/ods-components';
+import Topbar from '@/components/topBar/TopBar.component';
+import { MessageList } from '@/components/message/MessageList.component';
 
 export default function Ssl() {
-  const [message, setMessage] = useState({ status: null, label: '' });
-
   return (
     <React.Suspense>
```

Comment:
keep in mind that without a fallback you will have a blank screen on this section when lazy loaded modules are loading